pybikes/bysykkel.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+import json
+
+from .base import BikeShareSystem, BikeShareStation
+from . import utils
+
+
+class BySykkel(BikeShareSystem):
+
+    authed = True
+
+    meta = {
+        'system': 'BySykkel',
+        'company': ['Urban Infrastructure Partner']
+    }
+
+    def __init__(self, tag, meta, feed_url, feed_details_url, key):
+        super(BySykkel, self).__init__(tag, meta)
+        self.feed_url = feed_url
+        self.feed_details_url = feed_details_url
+        self.key = key
+
+    def update(self, scraper=None):
+        if scraper is None:
+            scraper = utils.PyBikesScraper()
+            scraper.headers['Client-Identifier'] = self.key
```

Comment:
This line should go outside of the `None` condition, since we always want it, no matter what scraper object we receive. Sorry if this part is a bit unclear, but makes sense once systems use pybikes

```
scraper = scraper or utils.PyBikesScraper()
scraper.headers['Client-Identifier'] = self.key
```